satpy/readers/__init__.py

```diff
@@ -150,9 +148,8 @@ def _assign_files_to_readers(files_to_sort, reader_names, ppp_config_dir,
 
 
 def _get_file_keys_for_reader_files(reader_files, group_keys=None):
-    """From a mapping from _assign_files_to_readers, get file keys.
-
-    Given a mapping where each key is a reader name and each value is a
+    """From a mapping from _assign_files_to_readers, get file keys."""
+    """Given a mapping where each key is a reader name and each value is a
```

Comment:
Not sure what happened with these docstrings, but could you revert these changes (this one and the two above)? I've never seen this formatting for docstrings but I know the docstrings were formatted properly already.

Reply:
Sure, the only reason I changed this is because the pre-commit hook kept flagging these lines with "no blank lines allowed after function docstring"

Reply:
This means that you are not allowed to have any blank lines between the docstring's last """ and the code iiuc